deflect/NetworkListener.h

```diff
@@ -82,12 +83,11 @@ public slots:
     void onEventRegistrationReply( QString uri, bool success );
 
 private:
+    detail::NetworkListener* _impl;
```

Comment:
+2 for the code review ( but we should be clear "I mean we all agree" about this prefix, postfix issues at a point, because it is creating complexity between styles, where here prefix is breaking coding style on a very big project )